src/scenes/PosterPage/PosterPage.tsx

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+import Page from 'components/core/Page/Page';
+import { BaseM, BaseXL, TitleM } from 'components/core/Text/Text';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import Button from 'components/core/Button/Button';
+import { contentSize, pageGutter } from 'components/core/breakpoints';
+import colors from 'components/core/colors';
+import PosterFigmaFrame from './PosterFigmaFrame';
+import ActionText from 'components/core/ActionText/ActionText';
+import StyledBackLink from 'components/NavbarBackLink/NavbarBackLink';
+import { useIsMobileWindowWidth } from 'hooks/useWindowSize';
+import { useToastActions } from 'contexts/toast/ToastContext';
+import useTimer from 'hooks/useTimer';
+
+export default function PosterPage() {
+  const isMobile = useIsMobileWindowWidth();
+  const { pushToast } = useToastActions();
+
+  const FIGMA_URL = 'https://www.figma.com/file/YrjZkebwPMIPr3aFx8o2w1/Untitled?node-id=1%3A2';
+
+  const { days, hours, minutes, seconds, hasEnded } = useTimer();
+
+  const countdown = `${days}:${hours}:${minutes}:${seconds}`;
+
+  const handleBackClick = () => {
+    // TODO: Replace with hook
+    window.history.back();
+  };
+
+  const handleSignPoster = () => {
+    pushToast(
+      'Thank you for participating in the (Object 006) 2022 Community Poster event.',
+      () => {},
+      true
+    );
```

Comment:
nit: the variables being passed in here are a bit hard to understand, we could update it to accept an object instead:

```js
pushToast({
  message: 'Thank you for participating in the (Object 006) 2022 Community Poster event.',
  onDismiss: () => {},
  autoClose: true,
})
```

would need to refactor other usages though

Reply:
Nice! This one much more readable. I can do follow up PR on this since I might need to check other places too